docs/advanced_topics/api/v2/usage.md

```diff
@@ -332,6 +332,20 @@ the current page back to the site's root page.
 The `?descendant_of` filter takes the id of a page and filter the list
 to only include descendants of that page (children, grandchildren etc.).
 
+
+### Filtering pages by site
+
+The `?site=` filter is used to filter the listing to only include pages that 
```

Comment:
I think it's worth mentioning here that by default, the API will look for the site based on the hostname of the request